addons/binding/org.openhab.binding.harmonyhub/src/main/java/org/openhab/binding/harmonyhub/handler/HarmonyHubHandler.java

```diff
@@ -209,7 +216,26 @@ private synchronized void connect() {
         discoTime = config.discoveryTimeout > 0 ? config.discoveryTimeout : DISCO_TIME;
         heartBeatInterval = config.heartBeatInterval > 0 ? config.heartBeatInterval : HEARTBEAT_INTERVAL;
 
-        discovery = new HarmonyHubDiscovery(discoTime);
+        String host = config.host;
+
+        // earlier versions required a name and used network discovery to find the hub and retrieve the host,
```

Comment:
You can also login without a session nowadays. I've modified the library. It works on the commandline just fine. 

See these changes:
* https://github.com/tuck182/harmony-java-client/compare/master...wborn:update-deps-and-fix-auth#diff-f31f90e97d792e1c39791b4cd693630eL37
* https://github.com/wborn/harmony-java-client/blob/update-deps-and-fix-auth/src/main/java/net/whistlingfish/harmony/protocol/MessageAuth.java#L33